src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -967,18 +966,20 @@ public void verifyContainsElement(By by) {
      * The check is done multiple times with waiting times in between to account for
      * timing issues due to page load, inconsistencies in Selenium API, etc.
      */
-    public void waitForTextsForAllStatusMessagesToUserEquals(final String... expectedTexts) {
-        Assumption.assertFalse(expectedTexts.length == 0);
+    public void waitForTextsForAllStatusMessagesToUserEquals(final String firstExpectedText,
+            final String... remainingExpectedTexts) {
+        List<String> expectedTexts = new ArrayList<String>(Arrays.asList(remainingExpectedTexts));
```

Comment:
There is a performance concern here, and we can also make use of Guava:

`Arrays.asList(ObjectArrays.concat(firstExpectedText, remainingExpectedTexts))`